packages/styles/scss/components/image/_image.scss

```diff
@@ -19,6 +19,10 @@
     display: block;
   }
 
+  .#{$prefix}--image__img--border {
+    outline: 1px solid $decorative-01;
```

Comment:
one more question: why `outline` instead `border`?

Reply:
Great question! I elected to use `outline` since it doesn't affect the dimensions of the rendered images. `border` would compress the images 2px horizontally and 2px vertically (1px on each edge).